Test/Integration/ImportExistingCategoryTest.php

```diff
@@ -0,0 +1,131 @@
+    protected $apiCategories;
+    protected $categoryFactory;
+    protected $categoryCollectionFactory;
+    protected $categoryRepository;
+    protected $consumer;
+    protected $descriptionHelper;
+
+    protected function setUp(): void
+    {
+        $objectManager = new ObjectManager($this);
+        $this->categoryCollectionFactory = Bootstrap::getObjectManager()->create(CollectionFactory::class);
+        $this->categoryFactory = Bootstrap::getObjectManager()->create(CategoryFactory::class);
+        $this->orderApiClientMock = $this->createMock(\StoreKeeper\StoreKeeper\Api\OrderApiClient::class);
+        $this->categoryRepository = Bootstrap::getObjectManager()->create(CategoryRepository::class);
+        $this->descriptionHelper = Bootstrap::getObjectManager()->create(DescriptionHelper::class);
+
+
+        $this->orderApiClientMock->method('listTranslatedCategoryForHooks')
+            ->willReturn($this->getCategoryImportData());
+
+        $this->apiCategories = $objectManager->getObject(
+            ApiCategories::class,
+            [
+                'orderApiClient' => $this->orderApiClientMock,
+                'categoryCollectionFactory' => $this->categoryCollectionFactory,
+                'categoryRepository' => $this->categoryRepository,
+                'descriptionHelper' => $this->descriptionHelper
+            ]
+        );
+
+        $this->consumer = $objectManager->getObject(
+            Consumer::class,
+            [
+                'categoriesHelper' => $this->apiCategories
+            ]
+        );
+    }
+
+    /**
+     * Feed queue consumer with category event "created" and verify that chanes applied to magento category
+     *
+     * @magentoConfigFixture current_store storekeeper_general/general/storekeeper_shop_language nl
+     * @magentoConfigFixture current_store general/locale/code nl_NL
+     * @magentoDataFixture StoreKeeper_StoreKeeper::Test/Integration/_files/categories.php
+     */
+    public function testProcess()
+    {
+        //Send webhook data to consumer
+        $this->consumer->process(self::WEBHOOK_DATA);
+
+        //Reload updated category from repository
+        $category = $this->categoryFactory->create()->loadByAttribute('url_key', $this->getStoreKeeperTestCategorySlug());
+        $categoryName = $category->getName();
+        $categoryMetaTitle = $category->getMetaTitle();
+        $categoryMetaKeywords = $category->getMetaKeywords();
+        $categoryMetaDescription = $category->getMetaDescription();
+
+        //Assert changes to category data
+        $this->assertEquals($categoryName, $this->getStoreKeeperTestCategoryTitle());
+        $this->assertEquals($categoryMetaTitle, $this->getStoreKeeperTestCategorySeoTitle());
+        $this->assertEquals($categoryMetaKeywords, $this->getStoreKeeperTestCategorySeoKeywords());
+        $this->assertEquals($categoryMetaDescription, $this->getStoreKeeperTestCategorySeoDescription());
```

Comment:
```suggestion
        $this->assertEquals($this->getStoreKeeperTestCategoryTitle(), $categoryName);
        $this->assertEquals($this->getStoreKeeperTestCategorySeoTitle(),$categoryMetaTitle);
        $this->assertEquals($this->getStoreKeeperTestCategorySeoKeywords(),$categoryMetaKeywords);
        $this->assertEquals($this->getStoreKeeperTestCategorySeoDescription(),$categoryMetaDescription);
```

Expect is first argument. Impacts the error message.